src/server/service/graphql/schema/mutations/customer_invoice/line/mod.rs

```diff
@@ -35,6 +35,20 @@ impl StockLineAlreadyExistsInInvoice {
     }
 }
 
+pub struct NotEnoughStockForReduction(pub String);
+#[Object]
+impl NotEnoughStockForReduction {
+    pub async fn description(&self) -> &'static str {
+        "Not enought stock for reduction"
```

Comment:
```suggestion
        "Not enough stock for reduction"
```

Minor nit: I prefer "Insufficient stock".